src/views/assets/const.js

```diff
@@ -125,6 +119,9 @@ export const assetFieldsMeta = (vm, platformType) => {
         ajax: {
           url: '/api/v1/assets/domains/'
         }
+      },
+      hidden: () => {
+        return vm.platform.domain_enabled === false
       }
     },
     accounts: {
```

Comment:
No issues were found after examining the given code. It was thoroughly checked across both versions and it appears to be functioning correctly. The logic seems to be intact but may require some cleanup based on best coding practices:

```js
export const filterSelectValues = (values) => {
}
```

```JS
function updatePlatformProtocols(vm, platformType) {}
```